src/types/mod.rs

```diff
@@ -508,10 +533,11 @@ impl<CL: Object> JsFunction<CL> {
               AS: IntoIterator<Item=Handle<'b, A>>
     {
         let mut args = args.into_iter().collect::<Vec<_>>();
-        let (isolate, argc, argv) = unsafe { prepare_call(cx, &mut args) }?;
+        let (argc, argv) = unsafe { prepare_call(cx, &mut args) }?;
+        let env = cx.env().to_raw();
         build(|out| {
             unsafe {
-                neon_runtime::fun::construct(out, isolate, self.to_raw(), argc, argv)
+                neon_runtime::fun::construct(out, env, self.to_raw(), argc, argv)
```

Comment:
FYI, most of the `isolate` -> `env` changes were mechanically done with find/replace to purge the term `Isolate` from `neon`.